src/bin/rdcore/unique_fs.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::cmdline::*;
+use anyhow::{bail, Result};
+
+use libcoreinst::blockdev::*;
+
+pub fn verify_unique_fs(config: &VerifyUniqueFsLabelConfig) -> Result<()> {
+    if let Ok(pts) = get_filesystems_with_label(&config.label) {
```

Comment:
Why are we dropping errors here?  Worth a comment.

Also, I think in *most* places we drop errors, we should at least log them (even if at a debug level).